### ISSUE TITLE
Working within eclipse?

### ISSUE THREAD

**Issue by rleibman** [akhikhl/wuff#46]
How does everyone work within eclipse?

We have a bunch of projects/features/plugins/apps. Our developers have so far been working within eclipse. Now I have to make all of these build with jenkins/gradle, without breaking everybody's work.
- wuff wants things (plugin.xml, META-INF, icons) in src/main/resources, but the way eclipse works it wants those at the root of the project
- Even if I leave things as they are, my extensions inside MANIFEST (e.g. Bundle-Version: ${project.ext.calculatedVersion}) break the way eclipse works and won't let me start the plugin as an application: "invalid version "${project.ext.calculatedVersion}": non-numeric"

Basically, I want to take a raw "Hello world" plugin project and make it build with wuff, but still work within eclipse.
